lib/watir/attribute_helper.rb

```diff
@@ -38,6 +38,10 @@ def attribute_list
     #  @return [$1] value of $3 property
     #
     def attribute(type, method, attr)
+      if %i(thead tfoot).include? method
```

Comment:
I tried using `#ignored_attributes` first. It is only called in the cleanup step, and it didn't seem to do anything.